src/components/command-bar/commands/search/helpers/generate-suggested-pages.tsx

```diff
@@ -63,6 +64,16 @@ const generateBasicSuggestedPages = (productSlug: ProductSlug) => {
 		return pages.filter((page) => page.url !== '/hcp/downloads')
 	}
 
+	const hasIntegrationsEnabled =
+		__config.dev_dot.product_slugs_with_integrations.includes(productSlug)
+	if (hasIntegrationsEnabled) {
+		pages.push({
+			icon: <IconPipeline16 />,
+			text: `${productSlugsToNames[productSlug]} Integrations`,
+			url: `/${productSlug}/integrations`,
+		})
+	}
+
```

Comment:
Ooo I need to make a TODO for myself to refactor this to use the [helpers in this file](https://github.com/hashicorp/dev-portal/pull/1604/files#r1093562661)

Reply:
Sorry, just refactor the `hasIntegrationsEnabled` check, not the whole block :smile